Problem1.java

```diff
@@ -0,0 +1,24 @@
+// Time Complexity :O(n) --> beacuse of for loop
+// Space Complexity :O(n)  ---> because of Hashmap
+import java.util.HashMap;
+
+public class problem1 {
+    public static int[] findindex(int[] nums,int target){
+        if(nums.length == 0 || nums.length == 1)
+            return new int[]{-1,-1};
+        HashMap<Integer,Integer> map = new HashMap<>();
+        for(int i = 0;i<nums.length;i++){
+            if(map.containsKey(target-nums[i]) && i != map.get(target-nums[i]))

```

Comment:
i != map.get(target-nums[i]) 
This condition will always be True.
Dont you think so ?